### PR TITLE
Implemented 2017 OI Panel buttons.

### PR DESCRIPTION
**Pull request by SethNguyenXBOT**

### CODE REVIEW

File: Competition2017/src/competition/operator_interface/OperatorInterface.java [hunk 1]
```diff
@@ -17,12 +17,14 @@
 public class OperatorInterface {
     public XJoystick leftJoystick;
     public XJoystick rightJoystick;
+    public XJoystick operatorJoystick;
```
Comment (aschokking) [inline Competition2017/src/competition/operator_interface/OperatorInterface.java]: let's call it operatorPanelJoystick just to be extra explicit

File: Competition2017/src/competition/operator_interface/OperatorInterface.java [hunk 2]
```diff
@@ -17,12 +17,14 @@
...
     
     public XXboxController controller;
   
     public JoystickButtonManager leftButtons;
     public JoystickButtonManager rightButtons;
-
+    public JoystickButtonManager operatorButtons;
```
Comment (aschokking) [inline Competition2017/src/competition/operator_interface/OperatorInterface.java]: and operatorPanelButtons

File: Competition2017/src/competition/operator_interface/OperatorCommandMap.java [hunk 3]
```diff
@@ -99,7 +103,7 @@ public void setupShiftCommand(
            ShiftGearCommand shiftLow, 
            ShiftGearCommand shiftHigh)
    {
-       shiftLow.setGear(Gear.LOW_GEAR);
+       shiftLow.setGear(Gear.LOW_GEAR);
```
Comment (WasabiFan) [inline Competition2017/src/competition/operator_interface/OperatorCommandMap.java]: Remove trailing whitespace

**Review by JohnGilb — APPROVED** [review]

**Review by JohnGilb — CHANGES_REQUESTED** [review]
Move the shoot command groups from `setupShooterWheelCommands()` into your new `setupShooterCommandGroup()`

**Review by JohnGilb — APPROVED** [review]

**Pull request merged**